packages/cozy-client/src/mock.js

```diff
@@ -19,7 +21,7 @@ const fillQueryInsideClient = (client, queryName, queryOptions) => {
   }
 }
 
-const mockedQueryFromMockedRemoteData = remoteData => qdef => {
+const mockedQueryFromMockedRemoteData = remoteData => async qdef => {
```

Comment:
why this async stuff?

Reply:
I go an type error on the 105 line when I'm using the function without jest